test/units/test_action_controller_request_proxy.rb

```diff
@@ -109,7 +109,7 @@ def test_that_proxy_simple_post_request_works_with_mixed_params
   def test_that_proxy_simple_put_request_works_with_mixed_params
     request_proxy = request_proxy(:put, {'key'=>'value'}, {'key2'=>'value2'})
 
-    expected_parameters = [["key", "value"]]
+    expected_parameters = [["key", "value"],["key2", "value2"]]
```

Comment:
Yeah I see what you mean, the original test looks wrong and this fixes it. LGTM.